Source/DafnyCore/DafnyOptions.cs

```diff
@@ -354,6 +354,8 @@ public enum IncludesModes {
     public bool UseStdin = false;
     public bool WarningsAsErrors = false;
     [CanBeNull] private TestGenerationOptions testGenOptions = null;
+    public List<string> CoverageReportsToMerge = new();
```

Comment:
Why do you add these fields here instead of a location that's specific to coverage?

Reply:
Done! I have moved them to `CoverageReportCommand.cs`